lobsterpy/cohp/test/test_describe.py

```diff
@@ -400,78 +438,146 @@ def test_text(self):
+                "Cs1 has a 18-fold coordination environment. It has 18 Cs-Cs (mean ICOHP: -0.49 eV, 18.741 "
+                "percent antibonding interaction below EFermi) bonds.",
                 "H2 has a 0 coordination environment. It has 0 bonds.",
             ],
         )
+        self.assertEqual(
+            self.describe_BaTiO3_orb.text,
+            [
+                "The compound BaTiO3 has 1 symmetry-independent cation(s) with relevant cation-anion interactions: Ti2.",
+                "Ti2 has an octahedral (CN=6) coordination environment. It has 6 Ti-O (mean ICOHP: -3.54 eV, "
+                "1.092 percent antibonding interaction below EFermi) bonds.",
+                "In the Ti-O bond, the maximum contribution is from the Ti(3dz2)-O(2pz) orbital, "
+                "contributing 25.5 percent, whereas the maximum antibonding contribution is from "
+                "the Ti(4s)-O(2s) orbital, contributing 100.0 percent.",
+            ],
+        )
+        self.assertEqual(
+            self.describe_C_orb.text,
+            [
+                "The compound C has 1 symmetry-independent atoms(s) with relevant bonds: C1.",
+                "C1 has a tetrahedral (CN=4) coordination environment. "
+                "It has 4 C-C (mean ICOHP: -9.59 eV, 0.0 percent antibonding interaction below EFermi) bonds.",
+                "In the C-C bond, the maximum contribution are from C(2py)-C(2s), C(2pz)-C(2s), C(2px)-C(2s), "
+                "C(2s)-C(2py), C(2s)-C(2pz), and C(2s)-C(2px) orbitals, contributing 7.94, 7.94, 7.94, 7.94, "
+                "7.94, and 7.94 percent, respectively, whereas the maximum antibonding contribution is from "
+                "the C(2s)-C(2s) orbital, contributing 20.477 percent.",
+            ],
+        )
+        self.assertEqual(
+            self.describe_NaSbF6_orb.text,
+            [
+                "The compound NaSbF6 has 3 symmetry-independent atoms(s) with relevant bonds: Na1, Sb2, F3.",
+                "Na1 has an octahedral (CN=6) coordination environment. It has 6 Na-F (mean ICOHP: -0.61 eV, "
+                "4.071 percent antibonding interaction below EFermi) bonds.",
+                "In the Na-F bond, the maximum contribution is from the Na(3s)-F(2s) orbital, contributing "
+                "77.48 percent, whereas the maximum antibonding contribution is from the Na(3s)-F(2s) orbital, "
+                "contributing 2.357 percent.",
+                "Sb2 has an octahedral (CN=6) coordination environment. It has 6 Sb-F (mean ICOHP: -5.45 eV, "
+                "0.0 percent antibonding interaction below EFermi) bonds.",
+                "In the Sb-F bond, the maximum contribution are from Sb(5pz)-F(2s), Sb(5py)-F(2s), "
+                "and Sb(5px)-F(2s) orbitals, contributing 39.12, 39.12, and 39.12 percent, respectively, "
+                "whereas the maximum antibonding contribution is from the Sb(5s)-F(2s) orbital, "
+                "contributing 35.799 percent.",
```

Comment:
It says: there is no antibonding contribution but then you list an orbital contribution for it.

Reply:
Yes, I also spotted this inconsistency yesterday. Trying an alternative approach for it. It happens because I did not use summed cohps of all relevant bonds , but rather looked relevant bond label to extract orbital contributions. Working on it since yesterday

Reply:
I tried with alternative approach, it will still happen and it makes sense also plot data supports it 😅

Reply:
Then you need to rephrase the text ;).  The sum is 0 but there are antibonding contributions from individual orbitals.

Reply:
![newplot(3)](https://github.com/JaGeo/LobsterPy/assets/91958822/da9e6269-0ffd-4766-846a-3fc26b44d244)

Reply:
Yes, but then the text is not describing it correctly. It's not a percentage of the 0.0% antibonding interactions. Make sure this is clear from the text.

Reply:
In general: what percentage are you giving there? orbital COHP/overall COHP? And then antibonding orbtia interactions/overall orbital interaction strength? Might also not be exactly clear.